### PR TITLE
Update App Archetype

### PR DESCRIPTION
**Pull request by JohnMcGuinness**
Updated with changes described in #247 

PTAL @jonathanaustin @ricksbrown

### CODE REVIEW

File: wcomponents-app-archetype/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -30,37 +31,68 @@
...
 
-		<dependency>
-			<scope>test</scope>
-			<groupId>com.github.bordertech.wcomponents</groupId>
-			<artifactId>wcomponents-test-lib</artifactId>
-			<version>${bordertech.wcomponents.version}</version>
-		</dependency>
+	<dependency>
+		<groupId>com.github.bordertech.wcomponents</groupId>
+		<artifactId>wcomponents-core</artifactId>
```
Comment (jonathanaustin) [inline wcomponents-app-archetype/src/main/resources/archetype-resources/pom.xml]: Fixes look good.

You could use the new "wcomponents-bundle" module, that brings in both "core" and the "theme".

**Comment by JohnMcGuinness** [comment]
@jonathanaustin made the suggested changes.

**Comment by jonathanaustin** [comment]
LGTM. If you update the branch I will merge the request.

**Comment by JohnMcGuinness** [comment]
@jonathanaustin updated PTAL

**Pull request merged**